lib/connectors/crawler/scheduler.rb

```diff
@@ -21,6 +21,12 @@ def connector_settings
         Utility::ExceptionTracking.log_exception(e, 'Could not retrieve Crawler connectors due to unexpected error.')
         []
       end
+
+      private
+
+      def connector_registered?(service_type)
```

Comment:
I can't see where this method is used.

Reply:
It's used in `sync_triggered?`:
https://github.com/elastic/connectors-ruby/blob/3d172ac61b04e6c78d6dd6d515037ad14f883bbb/lib/core/scheduler.rb#L62

`heartbeat_triggered?`:
https://github.com/elastic/connectors-ruby/blob/3d172ac61b04e6c78d6dd6d515037ad14f883bbb/lib/core/scheduler.rb#L129

and `configuration_triggered?`:
https://github.com/elastic/connectors-ruby/blob/3d172ac61b04e6c78d6dd6d515037ad14f883bbb/lib/core/scheduler.rb#L144

It override the method in `Core:Scheduler`:
https://github.com/elastic/connectors-ruby/blob/3d172ac61b04e6c78d6dd6d515037ad14f883bbb/lib/core/scheduler.rb#L149-L156

So for crawler, this method returns `true` as long as it's a crawler connector.

Reply:
Got it, thank you!